pipeline/frontend/yaml/constraint/constraint.go

```diff
@@ -195,7 +195,8 @@ func (c *Constraint) Match(m metadata.Metadata, global bool, env map[string]stri
 		if err != nil {
 			return false, err
 		}
-		match = match && result.(bool)
+		bresult, _ := result.(bool)
```

Comment:
can result be something beside bool?

Reply:
Dont know I just added error handling for this assertion as well.